<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QTextEdit" name="chatBox"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="portEdit"/>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="fileButton">
      <property name="text">
       <string>发送文件</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="hostEdit"/>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="sendButton">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="connectButton">
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" rowspan="2">
     <widget class="QLineEdit" name="messageEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>portEdit</tabstop>
  <tabstop>hostEdit</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>chatBox</tabstop>
  <tabstop>messageEdit</tabstop>
  <tabstop>sendButton</tabstop>
  <tabstop>fileButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
